From 3419eb5955c0ae7320312a9f8184fc20939ef0d3 Mon Sep 17 00:00:00 2001
From: Isuru Fernando <isuruf@gmail.com>
Date: Sat, 17 Oct 2020 22:37:44 -0500
Subject: [PATCH 2/2] don't set -arch

---
 bootloader/wscript | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/bootloader/wscript b/bootloader/wscript
index ae1d133..35c882b 100644
--- a/bootloader/wscript
+++ b/bootloader/wscript
@@ -366,9 +366,6 @@ def set_arch_flags(ctx):
                 mac_arch = UNIVERSAL2_FLAGS
             else:
                 mac_arch = ['-arch', 'x86_64']
-        ctx.env.append_value('CFLAGS', mac_arch)
-        ctx.env.append_value('CXXFLAGS', mac_arch)
-        ctx.env.append_value('LINKFLAGS', mac_arch)
 
     # AIX specific flags
     elif ctx.env.DEST_OS == 'aix':
-- 
2.31.1

