From c58f7a6aab049a7f69c38fb52276d6eeee43ae13 Mon Sep 17 00:00:00 2001
From: Isuru Fernando <isuruf@gmail.com>
Date: Sat, 17 Oct 2020 22:37:44 -0500
Subject: [PATCH] don't set -arch

---
 bootloader/wscript | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/bootloader/wscript b/bootloader/wscript
index 4cf059a..561932b 100644
--- a/bootloader/wscript
+++ b/bootloader/wscript
@@ -324,9 +324,6 @@ def set_arch_flags(ctx):
             mac_arch = ['-arch', 'i386']
         else:
             mac_arch = ['-arch', 'x86_64']
-        ctx.env.append_value('CFLAGS', mac_arch)
-        ctx.env.append_value('CXXFLAGS', mac_arch)
-        ctx.env.append_value('LINKFLAGS', mac_arch)
 
     # AIX specific flags
     elif ctx.env.DEST_OS == 'aix':
-- 
2.30.2

