From 4530fc1be2d139935a08b28c879c0edcd9b8045f Mon Sep 17 00:00:00 2001
From: Isuru Fernando <isuruf@gmail.com>
Date: Sat, 17 Oct 2020 22:37:44 -0500
Subject: [PATCH 1/2] don't set -arch

---
 bootloader/wscript | 2 --
 1 file changed, 2 deletions(-)

diff --git a/bootloader/wscript b/bootloader/wscript
index 8795fe5a0..0c7912ceb 100644
--- a/bootloader/wscript
+++ b/bootloader/wscript
@@ -320,8 +320,6 @@ def check_arch_cflag(cflag32, cflag64):
             else:
                 # Default to whatever the compiler is configured to build.
                 mac_arch = []
-        ctx.env.append_value('CFLAGS', mac_arch)
-        ctx.env.append_value('LINKFLAGS', mac_arch)
 
     # AIX specific flags
     elif ctx.env.DEST_OS == 'aix':
-- 
2.46.1

