From cb8b11e8c95d6e7d4eae7f4dc1436420671cf647 Mon Sep 17 00:00:00 2001
From: Nehal J Wani <nehaljw.kkd1@gmail.com>
Date: Sun, 11 Aug 2024 22:53:11 +0100
Subject: [PATCH 2/2] Specify missing header for _NSGetExecutablePath

---
 bootloader/src/pyi_main.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/bootloader/src/pyi_main.c b/bootloader/src/pyi_main.c
index 008896613..1edf8cb56 100644
--- a/bootloader/src/pyi_main.c
+++ b/bootloader/src/pyi_main.c
@@ -41,6 +41,10 @@
     #include <Carbon/Carbon.h>  /* TransformProcessType */
 #endif
 
+#if defined(__APPLE__)
+    #include <mach-o/dyld.h>  /* _NSGetExecutablePath() */
+#endif
+
 /* PyInstaller headers. */
 #include "pyi_main.h"
 #include "pyi_global.h"  /* PYI_PATH_MAX */
-- 
2.33.1

